client/my-sites/site-settings/media-settings/style.scss

```diff
@@ -0,0 +1,14 @@
+.media-settings__module-settings.is-indented {
+	margin-left: 32px;
+	margin-top: 16px;
+
+	.form-label {
```

Comment:
Most of these styles can be removed if you use `.site-settings__module-settings` and `.site-settings__child-settings` that were introduced in #11240 - we don't need to duplicate these styles.